autoware_launch/config/perception/object_recognition/prediction/map_based_prediction.param.yaml

```diff
@@ -0,0 +1,28 @@
+/**:
+  ros__parameters:
+    enable_delay_compensation: true
+    prediction_time_horizon: 7.8 #[s]
```

Comment:
@ismetatabay
Hi. I found this PR changed prediction_time_horizon to 7.8 sec (original was 10.0 sec).

Is this an intended change? If you don't mind, can you tell me why you chose this value? (For us, it is more convenient to leave it at 10sec for now.)

Reply:
@YoshiRi Hi, It was a not intended change. I have no idea how its changed 10.0 to 7.8. I am creating PR now. Sorry for this.

Reply:
Here is the PR link: https://github.com/autowarefoundation/autoware_launch/pull/523